plugins/plugin-chart-table/src/utils/extent.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+export default function extent<T = number | string | Date | undefined | null>(values: T[]) {
```

Comment:
This is copied from `d3-array`, didn't want to bring in the whole library because the function is relatively small and I don't want new dependencies.

This is preferred than `Math.min` and `Math.max` because the fail on large arrays (~10^6).

Reply:
I think `d3-array` as dependency is ok. D3 was written with tree-shaking in mind so `import { extent } from 'd3-array';` should not cause much more weight.